containerd/containerd.go

```diff
@@ -41,7 +41,7 @@ func (d *Driver) pullImage(imageName string) (containerd.Image, error) {
 	return d.client.Pull(d.ctxContainerd, imageName, containerd.WithPullUnpack)
 }
 
-func (d *Driver) createContainer(image containerd.Image, containerName, containerSnapshotName, containerdRuntime, netnsPath, secretsDir, taskDir, allocDir string, env []string, memoryLimit int64, config *TaskConfig) (containerd.Container, error) {
+func (d *Driver) createContainer(image containerd.Image, containerName, containerSnapshotName, containerdRuntime, netnsPath, secretsDir, taskDir, allocDir string, env []string, memoryLimit, cpuShares int64, config *TaskConfig) (containerd.Container, error) {
```

Comment:
This is pretty long at this point...  should there be a config struct passed in, maybe?

Reply:
+1 but probably as part of a refactor rather than part of this change

Reply:
Good idea! Let me create a ContainerConfig struct and pass a pointer to the struct instead of passing individual fields.